source/content/guides/autopilot/02-enable-autopilot.md

```diff
@@ -53,7 +53,7 @@ To work with Autopilot, [switch to the Workspace](/guides/new-dashboard/workspac
 
    Click **Continue** to choose pages for screenshot comparison tests.
 
-1. Add pages to track for visual regression testing:
+1. Add pages to track for visual regression testing. Autopilot automatically suggests up to ten URL paths during setup:
 
   ![Autopilot Setup - Visual Review screen. Select which pages should be covered by visual regression testing.](../../../images/autopilot/autopilot-setup-visual-review.png)
```

Comment:
@EdwardAngert - is it just me, or is this image just the _slightest_ bit blurry?

Reply:
it looks okay on the preview- but it very well might be because of the imageoptim

is it still blurry in the [preview](https://pr-6654--pantheon-docs.my.pantheonfrontend.website/docs/guides/autopilot/enable-autopilot/)?

Reply:
Ah- it looks the same as the others 👌🏾